### PR TITLE
add: `typora-deb`

### PR DESCRIPTION
**Pull request by erenfro**
A minimal markdown editor and reader.

### CODE REVIEW

**Comment by erenfro** [comment]
Original typora deb file has:
Version: 1.9.3-1
Does this need to match up and reflect in the typora-deb pkgver, or is 1.9.3 fine for that?

**Comment by Elsie19** [comment]
It's fine as is.

File: packages/typora-deb/typora-deb.pacscript [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+arch=('amd64')
+license=('custom:"Copyright (c) 2015 Abner Lee All Rights Reserved."')
+sha256sums=("deb47f0af1458d13e4cf6ee69b55ade61c204549cb2fb9692c5280da6a082f1f")
+source=("https://download.typora.io/linux/${gives}_${pkgver}_amd64.deb")
+optdepends=(
+  "noto-fonts-enoji: Or some other emoji fonts to see/use emojis"
+  "pandoc: Import/export for extra file formats"
+)
+replaces=("typora")
+pkgdesc="A minimal markdown editor and reader."
```
Comment (Elsie19) [inline packages/typora-deb/typora-deb.pacscript:15]:
```suggestion
pkgdesc="A minimal markdown editor and reader"
```

File: packages/typora-deb/typora-deb.pacscript [hunk 2]
```diff
@@ -0,0 +1,18 @@
+pkgname="typora-deb"
+gives="typora"
+pkgver="1.9.3"
+repology=("project: typora")
+depends=("libgtk-3-0" "libnss3")
+arch=('amd64')
+license=('custom:"Copyright (c) 2015 Abner Lee All Rights Reserved."')
```
Comment (Elsie19) [inline packages/typora-deb/typora-deb.pacscript:7]: This has to change. It's not a valid license name.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Comment by Elsie19** [comment]
Match the deb version. `pkgver` should match exactly what is given in the deb's Version field.

File: packages/typora-deb/typora-deb.pacscript (hunk 2 above)
Comment (erenfro) [reply packages/typora-deb/typora-deb.pacscript:7]: How so? This is proprietary payware software. It's not using any SPDX type licensing but their own.

Comment (oklopfer) [reply packages/typora-deb/typora-deb.pacscript:8]: then you don't need to provide the license variable at all (and shouldn't). It is meant for licenses that are already commonly available to systems through the `spdx-licenses` package (which, now that I say this out loud, should be made a pacstall dependency for next version).

**Review by oklopfer — APPROVED** [review]

**Pull request merged**